unfetter-discover/unfetter-ui
# Issue 977/baseline summary

This now displays create date/time and last modified date/time.
It correctly detects the status of the published flag and displays a button to Publish (does not Publish yet) or the published date (uses last modified time). Additionally the mega menu should allow changing between Baselines.

To Test:
Go to the Baseline dashboard.
Should see create date and last modified in the left sidebar.
Should see Published date (same as last modified) in upper right.
Opening the Mega menu should display all baselines.
Clicking a baseline in the Mega menu should switch to that baseline (no edit yet).

If Published flag is not set, should see Publish button, but I don't think we can save yet via the UI, much less save as a draft.
